src/DeviceRenderer.js

```diff
@@ -299,7 +299,7 @@ module.exports = class DeviceRenderer {
     disconnect() {
         this.initialized = false;
 
-        if (typeof this.mediaManager !== 'undefined') {
+        if (this.mediaManager) {
             this.mediaManager.disconnect();
```

Comment:
Si ça peut être `undefined` tu devrais utiliser l'optional chaining pour appeler la méthode `disconnect`

Reply:
this.mediaManager && this.mediaManager.disconnect(); me semble plus lisible que this.mediaManager?.disconnect(); pour l'appel d'une fonction non ?